tmp/name_primer.test

```diff
@@ -1,6 +0,0 @@
-Agaricus campestras
```

Comment:
Don't remember why this file changed.  I think at some point (maybe after some combination of: migration, switching branches, trying to reset the db) Git refused to let me make commits unless I did something about the file.  I think I removed it.
Perhaps more important:  (a) why are these changes showing up? -- I thought everything in this directory was outside of version control; (b) do changes to files in this directory matter?